### PR TITLE
Fixes Syndicate Bomb logs

### LINKED ISSUE #20188: Syndicate bombs have weird and spammy logs
I didn't review the code so I can't provide a complete list, but wire cutting/pulsing shouldn't appear in explosion logs, especially when it isn't causing an explosion

image

countdown/timer logs are typically in the trigger log type, which syndicate bombs also use, but don't pass an actor to:
image

it also spams logs for the actual explosion. Here you see the generic explosion log, a seemingly unnecessary detonation log, and a second detonation log with an actor
image

iirc the generic explosion log automatically logs the actor though a parameter in the method that actually causes explosions, all 3 of those logs should be collapsed into that. All wire cut/pulse logs should be in the WireHacking log type. Nothing should be spamming out multiple logs that duplicate information:
image

### PR DESCRIPTION
**Pull request by LankLTE**
## About the PR

Closes #20188. Removes every copied / incorrectly typed log. 

## Why / Balance

Read the issue for context.

## Technical details

Nearly everything that had issue was already logged. Wirecutting / pulsing is automatically logged under WireHacking. The explosion was automatically logged as well but it didn't pass an actor, which is now fixed (passes the wirecutter if done manually, the activator if done via timer). Defusing still needs a manual log as this is the defusable system- but it is no longer duplicated (only the one that passes an actor remains).  

I have no idea if this is correct so scream at me if it isn't please

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Some pictures to hopefully help: 

(Messing with wires and setting off the boom wire)
image
(Letting the bomb timer go off)
image
(Defusing bomb)
image

### CODE REVIEW

File: Content.Server/Defusable/Systems/DefusableSystem.cs [hunk 1]
```diff
@@ -313,31 +294,23 @@ public void ActivateWirePulse(EntityUid user, Wire wire, DefusableComponent comp
...
     }
 
     public bool BoomWireCut(EntityUid user, Wire wire, DefusableComponent comp)
     {
         if (comp.Activated)
         {
-            EntityManager.System<DefusableSystem>().TryDetonateBomb(wire.Owner, comp);
-            _adminLogger.Add(LogType.Explosion, LogImpact.Extreme,
-                $"{ToPrettyString(user):user} cut the BOOM wire of {ToPrettyString(wire.Owner):entity} and caused it to detonate!");
+            EntityManager.System<DefusableSystem>().TryDetonateBomb(wire.Owner, user, comp);
```
Comment (metalgearsloth) [inline Content.Server/Defusable/Systems/DefusableSystem.cs:309]: Isn't this already in defusablesystem?

File: Content.Server/Defusable/Systems/DefusableSystem.cs [hunk 2]
```diff
@@ -313,31 +294,23 @@ public void ActivateWirePulse(EntityUid user, Wire wire, DefusableComponent comp
...
         }
         else
         {
             EntityManager.System<DefusableSystem>().SetUsable(comp, false);
```
Comment (metalgearsloth) [inline Content.Server/Defusable/Systems/DefusableSystem.cs:313]: Same here.

File: Content.Server/Defusable/Systems/DefusableSystem.cs (hunk 1 above)
Comment (LankLTE) [reply Content.Server/Defusable/Systems/DefusableSystem.cs:309]: sloth this is defusablesystem

File: Content.Server/Defusable/Systems/DefusableSystem.cs (hunk 2 above)
Comment (LankLTE) [reply Content.Server/Defusable/Systems/DefusableSystem.cs:313]: same here

File: Content.Server/Defusable/Systems/DefusableSystem.cs (hunk 1 above)
Comment (LankLTE) [reply Content.Server/Defusable/Systems/DefusableSystem.cs:309]: I am dumb

**Pull request merged**